hack/make-rules/test-cmd-util.sh

```diff
@@ -1440,16 +1440,23 @@ kube::util::non_native_resources() {
 run_non_native_resource_tests() {
   kube::util::non_native_resources
 
-  # Test that we can list this new third party resource (foos)
+  # Set spec field for CRDs
+  if [ $1 == CRD ]; then
+    SPEC=spec.
+  else
```

Comment:
would leave it. makes it clearer. Maybe add to the comment: "For TRPs this is the empty string because TPRs don't have spec and status, but are flat."